.travis.yml

```diff
@@ -3,7 +3,6 @@ script: rake rspec
 rvm:
   - 2.1.4
```

Comment:
We should validate this list of ruby versions against what's being used in production environments. Ideally, we'd test with ruby 2.2, and/or upgrade applications using Ruby 2.0 to something newer.